### PR TITLE
fix breaking rqt_tree and update script installation

### PR DESCRIPTION
**Pull request by youliangtan**
Fixes on ros2 `rqt_tf_tree`.

- This fix is mainly related to https://github.com/ros-visualization/rqt_tf_tree/issues/32 . Alongside sneaking in a fix on https://github.com/ros-visualization/rqt_tf_tree/issues/29.

- Also, updated the script installation (setup.py) according to the convention as https://github.com/ros-visualization/rqt_console/tree/ros2

- If this works, would appreciate if `rqt_tf_tree` can be available as binary dist, as this is one of the most useful viz tool to have on ROS. https://github.com/ros-visualization/rqt_tf_tree/issues/31

Signed-off-by: youliang <tan_you_liang@hotmail.com>

### CODE REVIEW

**Comment by mabelzhang** [comment]
Thank you! What is the difference between `safe_load` (as in #30) and `full_load`, is one of them better?

**Comment by youliangtan** [comment]
Hmmm, good point. Although this is internal comm, `safe_load` is still preferred as it prevents execution of arbitrary code through config file serialization. I can change that.
Maybe I can remove the yaml loader line here in this pr? since this #30 can be merged into both ros1 and ros2 code base?

**Comment by mabelzhang** [comment]
Thanks, I think changing the `full_load` to `safe_load` in this PR would be good. #30 (now merged) is for ROS 1, this PR is for ROS 2.

**Review by mabelzhang — APPROVED** [review]
Thank you for the fix! I tested it on the latest ROS 2 development code and it worked well.
Just one minor change requested.

I can release the package after this is merged. Hopefully it works on Foxy and Galactic too, as I do not have the environments set up to test those :sweat_smile:

File: setup.py
Comment (youliangtan) [reply setup.py]: Updated! https://github.com/ros-visualization/rqt_tf_tree/pull/33/commits/60211467904f0d0e9743fa90a4c312317e05d368

**Comment by mabelzhang** [comment]
Release pending approvals of these PRs
https://github.com/ros/rosdistro/pull/31213
https://github.com/ros/rosdistro/pull/31214
and respective distro syncs. [Foxy](https://discourse.ros.org/t/preparing-for-foxy-sync-2021-11-04/22945/2) and [Galactic](https://discourse.ros.org/t/preparing-for-galactic-sync-2021-11-09/22949/2) have just been synced in the past week or two, so it'll probably be a month or so before they're out.

**Comment by Timple** [comment]
Releasing for `rolling` will automatically make this package available on new distributions.

And support folks running `rolling` :roller_skate:

**Pull request merged**
